bezier_application/package.xml

```diff
@@ -9,17 +9,18 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>roscpp</build_depend>
   <build_depend>bezier_library</build_depend>
+  <build_depend>moveit_ros_planning_interface</build_depend>
+  <build_depend>roscpp</build_depend>
+  <build_depend>rviz_visual_tools</build_depend>
   <build_depend>tf</build_depend>
   <build_depend>tf_conversions</build_depend>
   <build_depend>visualization_msgs</build_depend>
-  <build_depend>moveit_ros_planning_interface</build_depend>
-  <build_depend>rviz_visual_tools</build_depend>
 
   <build_export_depend>bezier_library</build_export_depend>
 
+  <exec_depend>bezier_library</exec_depend>
+  <exec_depend>fanuc_m20ia_moveit_config</exec_depend>
   <exec_depend>roscpp</exec_depend>
   <exec_depend>rviz_visual_tools</exec_depend>
```

Comment:
Could coalesce into a single `depend` (same for other deps that are both `build` and `run_depend`s.